cache/cache_test.go

```diff
@@ -76,10 +77,20 @@ func TestElements(t *testing.T) {
 
 		Convey("Given that I try to add or update an element in the cache, I should not get an error", func() {
 
-			c.AddOrUpdate(newid, secondValue)
+			ok := c.AddOrUpdate(newid, secondValue)
 			newvalue, err := c.Get(newid)
 			So(newvalue, ShouldEqual, secondValue)
 			So(err, ShouldBeNil)
+			So(ok, ShouldBeFalse)
+		})
+
+		Convey("Given that I try to update an element in the cache, I should not get an error", func() {
```

Comment:
fix the convey message .. .the intent of next convey message is to make sure it was an update.